### PR TITLE
Feature/refactor block part5

### PR DESCRIPTION
**Pull request by whaeck**
The photonuclear namespace is done in this PR.

ALso provides a few more changes due to review (the fluorescence JXS value).

### CODE REVIEW

File: src/ACEtk/PhotoatomicTable/src/generateData.hpp [hunk 1]
```diff
@@ -104,6 +104,13 @@ Data generateData( unsigned int z,
     jxs[3] = xss.size() + 1;
     xss.insert( xss.end(), jflo->begin(), jflo->end() );
   }
+  else {
+
+    if ( nepr > 0 ) {
+
+      jxs[3] = xss.size() + 1; // can't have zero in this case
```
Comment (cjosey) [inline src/ACEtk/PhotoatomicTable/src/generateData.hpp]: Hmm. This information is read for both the old and new data. I'll need to cross-check more datasets...

**Review by cjosey — DISMISSED** [review]

File: src/ACEtk/PhotoatomicTable/src/generateData.hpp (hunk 1 above)
Comment (cjosey) [reply src/ACEtk/PhotoatomicTable/src/generateData.hpp]: Yeah, I have to ask which dataset you saw that didn't have this value set? Every library I opened had it. MCPLIB, 02, 03, 04, 84.

Comment (whaeck) [reply src/ACEtk/PhotoatomicTable/src/generateData.hpp]: I thought mcplib had it set to 0, at least my test file had it like that: 
https://github.com/njoy/ACEtk/blob/feature/refactor-block-part5/src/ACEtk/PhotoatomicTable/test/resources/1000.01p
https://github.com/njoy/ACEtk/blob/feature/refactor-block-part5/src/ACEtk/PhotoatomicTable/test/resources/3000.03p

Comment (whaeck) [reply src/ACEtk/PhotoatomicTable/src/generateData.hpp]: I'll redownload the libraries just to check.

Comment (whaeck) [reply src/ACEtk/PhotoatomicTable/src/generateData.hpp]: You are correct, it has always been there

Comment (whaeck) [reply src/ACEtk/PhotoatomicTable/src/generateData.hpp]: @cjosey it's been changed so now everyone has it

**Review by cjosey — APPROVED** [review]

**Comment by whaeck** [comment]
I'll wait till tomorrow to merge. The github runners are a bit overwhelmed by our effectiveness today ;-)

**Pull request merged**